Modules/data.py

```diff
@@ -42,7 +42,7 @@
     462394160629153812: 2,
     699995169629667370: 2,
     379023623245004810: 2,
-    762530852977901622: 3, # -- replace with actual general_channel
+    306885827189800960: 3,

```

Comment:
Might have been a mistake on my part, but this is not the correct id fro the general channel.
The correct id is: ``306885827189800960``.